frontend/lib/queries/autogen/AllSessionInfo.graphql

```diff
@@ -1,11 +1,11 @@
 # This file was auto-generated by querybuilder, please do not edit it.
 fragment AllSessionInfo on SessionInfo {
   rentalHistoryInfo {
-    address,
-    borough,
     firstName,
     lastName,
     apartmentNumber,
+    address,
+    borough,
     phoneNumber,
     addressVerified
   },
```

Comment:
Similar to the `schema.json` changes, this is just happening because the order of the fields in the form changed.